scribus/styles/charstyle.attrdefs.cxx

```diff
@@ -19,7 +19,7 @@
 ATTRDEF(double, fontSize, FontSize, 200, true)
 ATTRDEF(double, fillShade, FillShade, 100, false)
 ATTRDEF(double, strokeShade, StrokeShade, 100, false)
-ATTRDEF(QString, fontFeatures, FontFeatures, "", false)
```

Comment:
The default font features should be an empty string. `liga` and `clig` are enabled in HarfBuzz by default, we don’t need to set them.